config/locales/en.yml

```diff
@@ -241,3 +241,218 @@ en:
               - "I cannot leave home because I think I’m at risk of severe illness from coronavirus"
               - "I cannot go out because I have a disability"
               - "I’m unable to leave my home for another reason"
+  results_link:
```

Comment:
Just taken a look at the logic for this page.
So these are just an array of questions right?

We need to do a comparison against the groups /need-help-with choices. Could we scope the questions by group?

Reply:
We were thinking something like:

```ruby
if show_options.include?(session[:user_response_to_question])
  <some component-y thing to show each of the links>
end
```

Reply:
Ah but i think there's a two part show.

We show a group only if they've selected the right section in that checkbox question.

Then... if they've shown a group we can do checks for each question + link